section4/exercises/classProperties.js

```diff
@@ -50,8 +109,57 @@ Make sure to run the file with node in your command line.
   // Create 2 object instances and store in variables (passing in different arguments)
   // Log those variables to the console.
 
+// Nina note - for all classes, I created 3 object instances instead of 2.
 // Prompt 1: Dog
+class Dog {
+  constructor(breed, weight, age) {
+    this.breed = breed;
+    this.weightInPounds = weight;
+    this.age = age;
+  }
+}
+
+var dog1 = new Dog ('Rotweiler', 100, 5);
+var dog2 = new Dog ('German Shepherd', 84, 6);
+var dog3 = new Dog ('Poodle', 60, 4);
+
+console.log(dog1);
+console.log(dog2);
+console.log(dog3);
 
 // Prompt 2: Snack
+class Snack {
+  constructor (nutrition, color, type) {
+    this.nutritionValue = nutrition;
+    this.color = color;
+    this.type = type;
+  }
+}
+
+var apple = new Snack (95, 'red', 'fruit');
+var sourPatchKids = new Snack (0, 'multi-colored', 'sugar');
+var broccoli = new Snack (100, 'green', 'vegetable');
+
+console.log(apple);
+console.log(sourPatchKids);
+console.log(broccoli);
+
+// Prompt 3: Shirt
+
+class Shirt {
+  constructor (material, size, cost) {
+    this.material = material;
+    this.size = size;
+    this.costInDollars = cost;
+  }
+}
+
+var tank = new Shirt ('cotton', 'small', 15);
+var sweater = new Shirt ('wool', 'large', 40);
+var longSleeve = new Shirt ('cotton-poly', 'medium', 25);
+
+console.log(tank);
+console.log(sweater);
+console.log(longSleeve);
 
-// Prompt 3: Shirt
\ No newline at end of file
+// I had to run the 3 'Parts' seperately in node because node is reading that I am attempting each class three times.
```

Comment:
This is a good solution. You could also modify the class name slightly to force the whole file to run, like `Dog` and `Dog1`, etc